scripts/download_holopix50k.sh

```diff
@@ -1,10 +1,36 @@
 #!/bin/bash
 #
-# Holopix50k download script for Linux/MacOS. Also temporarily installs `gsutil` if not installed already.
+# Holopix50k download script for Linux/MacOS. Temporarily installs `gsutil` if not installed already.
 #
-
 set -e
-
+#
+#######################################
+# Cleans up gsutil installation after download.
+# Arguments:
+#   None
+# Outputs:
+#   Displays clean up steps.
+#######################################
+cleanup() {
```

Comment:
If I already have GSUtil installed, will this clean it up as well?

Reply:
Nope

Reply:
Cool that should be fine then